run_end_to_end_tests.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
+# Usage prepare_end_to_end_tests.sh -c config_file
+#                                   -a configuration_anchor
+#                                   -h security_server_host
+#                                   -n security_server_name
+#                                   -k private_key_file
+#                                   -u credentials
+#                                   -o output file
+#
+# Description of required command line arguments:
+#
+#   -c: config file
+#   -a: configuration-anchor file
+#   -h: host name or IP address of the security server
+#   -n: security server name
+#   -k: private ssh key file
+#   -u: credentials
+#   -o: output file
+#
+#
+# Usage example: prepare_end_to_end_tests.sh -c tests/resources/test-config.yaml
+#                                             -a /etc/xroad/configuration_anchor.xml
+#                                             -h ss
+#                                             -n ss
+#                                             -k /home/user/id_rsa
+#                                             -u xrd:secret
+#                                             -o tests/resources/test-config.yaml
```

Comment:
Is it intentional that input config file (``-c``) and output file (``-o``) are the same here?